scripts/ci/run-test-env-manager.sh

```diff
@@ -131,10 +131,10 @@ function download_from_remote {
     while read -r _path _type _download_url _url ; do
       if [[ "${_type}" == "file" ]] ; then
         echo curl "${_download_url}" --output "${OUTPUT_DIR}/${_path}"
-        curl "${_download_url}" --output "${OUTPUT_DIR}/${_path}" --silent
+        curl "${_download_url}" --output "${OUTPUT_DIR}/${_path}" --logtype silent
```

Comment:
```suggestion
        curl "${_download_url}" --output "${OUTPUT_DIR}/${_path}" --silent
```
This is a flag of `curl` and we should not change it.
Thx!